tools/system_libs.py

```diff
@@ -82,13 +83,12 @@ def clean_env():
   return safe_env
 
 
-def run_build_commands(commands):
+def run_build_commands(commands, build_dir=None):
   # Before running a set of build commands make sure the common sysroot
   # headers are installed.  This prevents each sub-process from attempting
   # to setup the sysroot itself.
   ensure_sysroot()
-  shared.run_multiple_processes(commands, env=clean_env())
-  logger.info('compiled %d inputs' % len(commands))
```

Comment:
This functions is also used to build ports (see `ports/__init__.py`) so maybe keep the logging in this function add add an options "num_inputs" argument?

Reply:
Sgtm.